requirements.txt

```diff
@@ -1,6 +1,6 @@
 pip
 docutils==0.16
-sphinx==4.3.2
+sphinx
```

Comment:
To minimize breakages, do you maybe want to specify a version after all?

The one that was downloaded was 7.32 according to https://github.com/ros-planning/moveit2_tutorials/actions/runs/7531539359/job/20500319203?pr=854#step:4:82

You can also use something like `sphinx>=5.0.0` if you just want to skirt around the immediate error but give users wiggle room when building locally.

Generally for these things it's good for the pip requirements to be pinned to actual versions and for any updates to be deliberate.